tests/smoke_tests/test_basic.py

```diff
@@ -809,3 +811,165 @@ def test_cli_exit_codes(generic_cloud: str):
         timeout=smoke_tests_utils.get_timeout(generic_cloud),
     )
     smoke_tests_utils.run_one_test(test)
+
+
+@pytest.mark.lambda_cloud
+@pytest.mark.skipif(not Lambda._check_compute_credentials()[0],
+                    reason='Lambda Cloud credentials not available')
```

Comment:
Is this special skipif mark required if `pytest.mark.lambda_cloud` is used above?